.github/workflows/integration_test.yaml

```diff
@@ -0,0 +1,13 @@
+name: Integration tests
+
+on:
+  pull_request:
+
+jobs:
+  integration-tests:
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    with:
+      extra-arguments: --cloud microk8s --jenkins-controller-name $(cat controller_name.txt) --jenkins-model-name $(cat model_name.txt) --jenkins-unit-number $(cat unit_number.txt)
```

Comment:
I'm impressed this works, I didn't spot it, where are those files created?

Reply:
The pre-run script `pre-integration-test.sh` calls `deploy_jenkins.py` which creates these files.
I was think about whether to add a comment about it.